doc/help-scoring.md

```diff
@@ -29,17 +29,19 @@ option.
 ## Popularity
 
 Popularity measures the number of apps that depend on a package over the past 60
-days. We show this on a normalized scale from 100% (the most used package) to 0%
-(the least used package), but are investigating if we can provide absolute usage
-counts in a future version.
+days. We show this as a percentile from 100% (among the top 1% of packages) to
```

Comment:
could we s/among the top 1% of packages/amount the top 1% most used packages/ ?

Reply:
Yeah that's more precise!